rllib/algorithms/impala/impala_learner.py

```diff
@@ -64,10 +65,30 @@ def build(self) -> None:
             )
         )
 
+        # Create and start the GPU-loader thread. It picks up train-ready batches from
+        # the "GPU-loader queue" and loads them to the GPU, then places the GPU batches
+        # on the "update queue" for the actual RLModule forward pass and loss
+        # computations.
+        self._gpu_loader_in_queue = queue.Queue()
+
         # Default is to have a learner thread.
         if not hasattr(self, "_learner_thread_in_queue"):
             self._learner_thread_in_queue = deque(maxlen=self.config.learner_queue_size)
 
+        # Create and start the GPU loader thread(s).
+        if self.config.num_gpus_per_learner > 0:
+            self._gpu_loader_threads = [
```

Comment:
Let's name the threads for better debugging.

Reply:
done